### PR TITLE
Redistribute CODEOWNERS

### PR DESCRIPTION
**Pull request by jmagman**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -81,23 +81,26 @@ packages/video_player/video_player_android/**            @camsim99
...
+packages/google_maps_flutter/google_maps_flutter_ios/**  @cbracken
+packages/google_sign_in/google_sign_in_ios/**            @vashworth @loic-sharma
+packages/image_picker/image_picker_ios/**                @vashworth @loic-sharma
+packages/image_picker/image_picker_macos/**              @jmagman
 packages/in_app_purchase/in_app_purchase_storekit/**     @louisehsu
 packages/ios_platform_images/**                          @jmagman
 packages/local_auth/local_auth_darwin/**                 @louisehsu
 packages/path_provider/path_provider_foundation/**       @jmagman
-packages/pigeon/**/ios/**/*                              @hellohuanlin
-packages/pointer_interceptor/pointer_interceptor_ios/**  @ditman
```
Comment (jmagman) [inline CODEOWNERS:95]: @LouiseHsu suggested she take this one, are you cool with that @ditman?

Comment (ditman) [reply CODEOWNERS:95]: 200% cool with it, thanks @LouiseHsu :)

**Review by ditman — APPROVED** [review]
LGTM!

**Review by loic-sharma — APPROVED** [review]

**Review by LouiseHsu — APPROVED** [review]
joining the lgtm train :)

**Pull request merged**